shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -723,6 +852,13 @@ - (void)setSelectedTextRangeLocal:(UITextRange*)selectedTextRange {
 - (void)setSelectedTextRange:(UITextRange*)selectedTextRange {
   [self setSelectedTextRangeLocal:selectedTextRange];
   [self updateEditingState];
+  if (_scribbleInProgress || _scribbleFocused) {
+    _scribbleFocused = false;
+    FlutterTextRange* flutterTextRange = (FlutterTextRange*)selectedTextRange;
```

Comment:
Add assert to cast.

Reply:
Done. I also found another place or two, so hopefully I've caught all of those (at least the ones in my changes).